### ISSUE TITLE
Build failure on Windows

### ISSUE THREAD

**Issue by star-hengxing** [OctoMap/octomap#418]
```
octomap\src\compare_octrees.cpp(136): error C2039: '_isnan': is not a member of 'std'
```

https://github.com/OctoMap/octomap/blob/863e069489ef619d7179e3b21480dffb41b2af66/octomap/src/compare_octrees.cpp#L42-L44

->

```
#if defined(_MSVC_LANG) && _MSVC_LANG < 201103L
#define isnan(x) _isnan(x)   
#endif 
```

**Comment by star-hengxing** [comment]
I found this https://github.com/OctoMap/octomap/pull/417

**Comment by ahornung** [comment]
Fixed with #417
